.travis/test-part.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# -*- firestarter: "shfmt -i 4 -ci -w %p" -*-
+
+set -euxo pipefail
+
+readonly test_part=${TEST_PART:-}
+
+case "$test_part" in
+Sol5)
+    npx buidler test --no-compile --config ./buidlerCoverageSol5.js
```

Comment:
why use coverage here?

can use "buildler.config.js" ??

Reply:
i just copy this one from tst.sh, will fix